package.json

```diff
@@ -7,9 +7,9 @@
     "node": "^6.0.0"
   },
   "scripts": {
-    "start": "bin/server",
-    "deploy": "bin/deploy",
-    "build": "node_modules/.bin/webpack"
+    "start": "node_modules/.bin/webpack-dev-server --inline --hot",
```

Comment:
Yeah, your're right. I don't see much use of `bin/server` after this. Nginx should be used as production server if we don't use GitHub Page or CDNs. There's no point to serve static content through node.